source/code/source/playfab/PlayFabApiSettings.cpp

```diff
@@ -5,39 +5,33 @@
 
 namespace PlayFab
 {
+    const std::map<std::string, std::string> PlayFabApiSettings::requestGetParams = {
```

Comment:
can this whole map be deleted and instead use a hardcoded string literal that gets appended on the URL?

Reply:
Not trivially. The requestParams are already designed using this pattern, and they're intended to be mutable: We have ours, customers could have theirs, and they need to be properly merged.
I'm willing to consider a future task to improve the performance and keep the mutability, but improve the performance.
Oh... const... bah, that const may not be correct...
I'll file the task.

Reply:
(Task filed)